src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -344,10 +344,10 @@ public async Task SendAsync_RequestHeaderInResponse_Success(string name, string
         }
 
         [OuterLoop("Uses external servers")]
+        [SkipOnPlatform(TestPlatforms.Browser, "Host is ignored on Browser")]
```

Comment:
What does this mean? Is `m.Headers.Host` problematic? Will it always be ignored on Browser or is this something that can be implemented to not be ignored?

Reply:
This is owned by underlying http implementation in browser. It could not be changed AFAIK.

Reply:
I'm not clear on why we are replacing the `ActiveIssue` with this.